VirtualList.svelte

```diff
@@ -25,6 +25,14 @@
 	let bottom = 0;
 	let average_height;
 
+	let previousContents = []
+	$: items, (()=>{
+		if (mounted && items.length != previousContents.length && items.length) {
+			keep = items
```

Comment:
What is this line `keep = items` for? I don't see `keep` defined or used elsewhere.

Reply:
That was my bad, it should have been from 

```
	let previousContents = []
	$: items, (()=>{
		if (mounted && items.length != previousContents.length && items.length) {
			previousContents = items
```

when I copied from my own code I made adjustments to make it more clear what I was doing and I must have forgotten to change that

I just commited a new change